frontend/packages/topology/src/data-transforms/transform-utils.ts

```diff
@@ -359,6 +359,12 @@ export const getBaseWatchedResources = (namespace: string): WatchK8sResources<an
       namespace,
       optional: true,
     },
+    services: {
+      isList: true,
+      kind: 'Service',
+      namespace,
+      optional: true,
+    },
```

Comment:
can we have this in knative plugin package?

Reply:
`Service` here is a base K8s Resource, so I think we can keep it here itself as we have d/dc/pods etc.

Reply:
make sense if in future this will be used by other use cases as well